.github/workflows/stale.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '15 22 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-pr-message: 'This PR is stale. Consider a pair-programming session to help move it forward.'
+        stale-pr-label: 'no-pr-activity'
+        close-pr-message: 'PR closed due to lack of activity'
+        days-before-stale: 30
```

Comment:
I'd like to hear from the team on what a good timeframe looks like, and if we want certain exceptions.

Reply:
I think 30 days is good 👍